netlify/edge-functions/og-image.tsx

```diff
@@ -0,0 +1,270 @@
+              fill="#007FFF"
+            />
+          </svg>
+          <div
+            style={{
+              height: 40,
+              width: 2,
+              backgroundColor: '#DAE2ED',
+              margin: '0 24px',
+            }}
+          />
+          <p
+            style={{
+              fontFamily: 'General Sans',
+              fontSize: '24px',
+              fontWeight: 600,
+              lineHeight: '40px',
+              color: '#007FFF',
+            }}
+          >
+            {product}
+          </p>
+        </div>
+        <div
+          style={{
+            fontFamily: 'General Sans',
+            fontStyle: 'normal',
+            fontWeight: 600,
+            fontSize: '68px',
+            lineHeight: '82px',
+            color: '#0B0D0E',
+            marginLeft: 100,
+            marginRight: 100,
+            marginTop: 200,
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'flex-start',
+          }}
+        >
+          {title &&
+            title.split('\\n').map((line) => (
+              <p style={{ margin: 0, flexWrap: 'wrap' }}>
+                {line.split('*').flatMap((text, index) => {
+                  if (index > 0) {
+                    starCount += 1;
+                  }
+
+                  const isBlue = starCount % 2 === 1;
+                  return text.split(' ').map((word) => (
+                    <span
+                      style={{
+                        color: isBlue ? '#007FFF' : '#0B0D0E',
+                        marginRight: word.length > 0 ? 15 : 0,
+                      }}
+                    >
+                      {word}
+                    </span>
+                  ));
+                })}
+              </p>
+            ))}
+        </div>
+
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'row',
+            flexWrap: 'wrap',
+            position: 'absolute',
+            bottom: 100,
+            left: 100,
+            right: 100,
+          }}
+        >
+          {authors &&
+            authors
```

Comment:
Would it be possible to render this whole div conditionally depending on whether there are authors or not? Just so the page's title is centered in case it's not a blog post (and thus without authors).